blog/_posts/2024-08-19-KAITO-finetuning.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Fine tune language models with KAITO on AKS"
+description: "Make your AI model even smarter, in just a few steps"
+date: 2024-08-19
+author: Sachi Desai
+categories: AI, Machine Learning, open-source, operator
```

Comment:
You need to pick a single category here. We need to add something for AI. I'll send details in Teams